tests/v2/test_waku_rln_relay.nim

```diff
@@ -631,6 +631,71 @@ suite "Waku rln relay":
                                  proof = proof)
     check:
       verified == false
+
+  test "invalidate messages with a valid, but stale root":
+    var rlnInstance = createRLNInstance()
+    check:
+      rlnInstance.isOk == true
+    var rln = rlnInstance.value
+
+    let
+      # create a membership key pair
+      memKeys = membershipKeyGen(rln).get()
+      # peer's index in the Merkle Tree
+      index = 5
+
+    # Create a Merkle tree with random members
+    for i in 0..10:
+      var member_is_added: bool = false
```

Comment:
Is there a reason (here and some places elsewhere) why `camelCase` is not used for variable naming?

Reply:
There's an issue which tracks some of the refactoring required - https://github.com/status-im/nwaku/issues/946.

I'll fix the code which was touched in the PR though.

Reply:
Oh right! Thanks. Didn't see that issue.

Reply:
Addressed in [a0b3761](https://github.com/status-im/nwaku/pull/1158/commits/a0b376199382ddf0174a2319187d5d5399ac4f1d)